packages/utils/interactive-hooks/src/useViewCommandFocus.win32.ts

```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
 import { findNodeHandle, UIManager } from 'react-native';
 import { IViewWin32 } from '@office-iss/react-native-win32';
-
-const setAndForwardRef = require('./setAndForwardRef');
+import { setAndForwardRef } from './setAndForwardRef';
```

Comment:
FWIW, I think this was still using `require()` because it was copied from RN Core

Reply:
Thanks for the fix!